src/openrct2/world/Footpath.cpp

```diff
@@ -2570,3 +2570,15 @@ bool PathElement::IsLevelCrossing(const CoordsXY& coords) const
 
     return ride->GetRideTypeDescriptor().HasFlag(RIDE_TYPE_FLAG_SUPPORTS_LEVEL_CROSSINGS);
 }
+
+bool PathElement::PathMustBeMadeInvisible(int32_t parkFileVersion)
```

Comment:
I think its better to move this to the importer code as a static function, this doesn't need to be exposed.